client/synapseJavaClient/src/main/java/org/sagebionetworks/client/SynapseClientImpl.java

```diff
@@ -317,6 +317,8 @@
 	private static final String MESSAGE_ORDER_BY_PARAM = "orderBy";
 	private static final String MESSAGE_DESCENDING_PARAM = "descending";
 
+	private static final String PORTAL_NOTIFICATION_SETTINGS_ENDPOINT = "https://www.synapse.org/#!SignedToken:Settings/";
```

Comment:
Migrate DEFAULT notification settings endpoint to the back end.  Client can override if desired.